website/content/en/docs/scorecard/custom-tests.md

```diff
@@ -0,0 +1,190 @@
+1. `image` - name and tag of the test image which was specified in the Makefile.
+2. `labels` - the name of the `test` and `suite` the test function belongs to. This can be specified in the `operator-sdk alpha scorecard` command to run the desired test.
+
+**Note**: `config.yaml` file need not necessarily be present in the same repository. The `operator-sdk alpha scorecard` command includes the `--config` flag using which the path to the config file can be provided.
+
+### Scorecard binary:
+
+The scorecard test image implementation requires the bundle under test to be present in the test image. The `GetBundle()` function reads the pod's bundle to fetch the manifests and scorecard configuration from the desired path.
+
+```Go
+	cfg, err := GetBundle("/bundle")
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+```
+
+The Scorecard binary is where the actual tests which are to be run are defined, and their names are passed as an argument. It is the entrypoint from where all the specified tests are run within a single test image.
+An example scorecard binary is present [here][].
+
+The names with which the tests are identified in `config.yaml` and would be passed in the `scorecard` command, are to be specified here.
+
+```Go
+...     
+switch entrypoint[0] {
+case tests.CustomTest1Name:
+    result = tests.CustomTest1(*cfg)
+    ...
+}
+...
+```
+
+The result of the custom tests which is in `scapiv1alpha2.ScorecardTestResult` format, is converted to json for the output.
+
+```Go
+prettyJSON, err := json.MarshalIndent(result, "", "    ")
+	if err != nil {
+		log.Fatal("failed to generate json", err)
+	}
+	fmt.Printf("%s\n", string(prettyJSON))
+```
+
+The names of the custom tests are also included in the `printValidTests()` function: 
+
+```Go
+func printValidTests() (result v1alpha2.ScorecardTestResult) {
+...
+    str := fmt.Sprintf("Valid tests for this image include: %s,
+            tests.CustomTest1Name
+    result.Errors = append(result.Errors, str")
+...
+
+```
+
+### Building the project
+
+The project makefile is to help us build the go project and the test image using docker. An example of the [makefile][smaple_makefile] script can be found in the sample test image.
+
+To build the project, use the `docker build` command and specify the desired name of the image in the format: `<repository_name>/<username>/<image_name>:tag`.
+
+Push the image to the remote repository by running the docker command:
+
+```
+docker push <repository_name>/<username>/<image_name>:tag
+```
+
+### Running scorecard command
+
+The `operator-sdk alpha scorecard` command is used to execute the scorecard tests by specifying the location of test bundle with `--bundle` flag. The name or suite of the tests which are to be executed can be specified with the `--selector` flag. The command will create scorecard pods with the image specified in `config.yaml` for the respective test. For example, the `CustomTest1Name` test provides the following json output.
```

Comment:
with the latest PR, the bundle image name or bundle path is now a positional argument instead of a command flag, --bundle is removed essentially and replaced with the positional argument to be consistent with other sdk commands.